### ISSUE TITLE
Add option to disable animations (aka: save our remote sessions)

### ISSUE THREAD

**Issue by drewboswell** [argoproj/argo-cd#5982]
# Summary

Add option to enable/disable GUI animations

# Motivation

When working remotely, animations can blow up and slow down your remote sessions du to variable frame rate overload. By opting into static status icons we can avoid sometimes significant (and frustrating) lag.

**Comment by warisp** [comment]
Agreed - even though the new version is very snappy the animations can make it a little slow over remote connections.

**Comment by jiribroulik** [comment]
Also for large applications with tens of resources it is quite slow to load the application page.

**Comment by alexmt** [comment]
[NodeUpdateAnimation](https://github.com/argoproj/argo-cd/blob/master/ui/src/app/applications/components/application-resource-tree/node-update-animation.tsx) is responsible for animation. I would suggest to support disabling it using user preferences: https://github.com/argoproj/argo-cd/blob/eb70a1e3fa4ffc7e479b75415217e62aa637ecc7/ui/src/app/shared/services/view-preferences-service.ts#L6

**Comment by shruthiiraju** [comment]
@alexmt / @drewboswell If nobody is working on this, and this is still a needed fix, I'd like to get started to implement this!

**Comment by alexmt** [comment]
Thank you @shruthiiraju !

**Comment by nagy** [comment]
Thank you @shruthiiraju for considering this implementation.

Can this please be (additionally) implemented using the css media query `prefers-reduced-motion`? This way, users dont have to disable animation for every application, but can set this systemwide.

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

**Comment by DanielJiang378** [comment]
@alexmt I've created a partial fix to this issue with possible follow ups.

**Comment by rowanmoul** [comment]
I absolutely support accessibility features so having the UI respond to a global media query is great, but the only animation that actually causes me grief is the network view's animated lines. With an application that deploys over 20 pods, and all their services and HttpRoute resources, there are probably a couple hundred lines on the screen and it can't even render them smoothly. It's constantly stuttering and hitching. When I disable the animation rule on `.application-resource-tree--network .application-resource-tree__line` my laptop's fans noticeably spin down.
